cmd/check.go

```diff
@@ -143,6 +143,14 @@ var checkCmd = &cobra.Command{
 				minVersion: "2.17.1",
 				docsURL:    "https://git-scm.com/book/en/v2/Getting-Started-Installing-Git",
 			},
+			{
+				name:       "Wget\t\t",
+				command:    "wget",
+				args:       []string{"--version"},
+				regexStr:   `^GNU Wget (0|[1-9]\d*)\.(0|[1-9]\d*)\.(0|[1-9]\d*)`,
+				minVersion: "1.20.3",
```

Comment:
looks like 1.20.3 is the latest one, people may not update their wget often
might be safer to go with some older versions as I doubt there's many breaking changes on wget

do you think it'll be good to allow some older versions too?

https://ftp.gnu.org/gnu/wget/?C=M;O=D

Reply:
We're all probably using and testing on`1.20.3` but I'm comfortable supporting a minor version back `1.19.0` released in 2017

Reply:
yeah 2012 definitely seems far back enough, I think as long as it covers most people's new computer cycle then its good 👍